docs/source/gbd2019_models/risk_effects/low_birthweight_short_gestation/index.rst

```diff
@@ -255,6 +255,57 @@ Restrictions
 Risk Exposure Categories and TMREL
 ++++++++++++++++++++++++++++++++++
 
+Here is a plot `created by Kjell <kjell_lbwsg_category_rr_notebook_>`_ that
+shows the LBWSG exposure categories and the mean relative risk estimate in each
+category (the mean is taken across all all sexes, age groups, and draws):
```

Comment:
@KjellSwedin Can you confirm that this description of your image of LBWSG categories with RRs is accurate?  (I.e. did you take the mean RR over all sexes, age groups, and draws?)

Reply:
Correct. The location is India.

Reply:
Great, thanks Kjell.  (As far as I can tell, the RRs for all locations are the same, but it might be worth my double-checking that...)